File: eRecipes/eRecipes.Model/Lajkovi.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class Lajkovi
    {
        public int LajkoviId { get; set; }

        public int KorisnikId { get; set; }

        public int ReceptId { get; set; }

        public DateTime DatumLajka { get; set; } = DateTime.Now;

        public virtual Korisnik Korisnik { get; set; } = null!;

        public virtual Recept Recept { get; set; } = null!;

    }
}

File: eRecipes/UI/erecipes_mobile/lib/models/lajkovi.dart
import 'package:erecipes_mobile/models/korisnik.dart';
import 'package:erecipes_mobile/models/recept.dart';
import 'package:json_annotation/json_annotation.dart';
part 'lajkovi.g.dart';

@JsonSerializable()
class Lajkovi {
  int? lajkoviId;
  int? korisnikId;
  int? receptId;
  DateTime? datumLajkanja;
  Korisnik? korisnik;
  Recept? recept;

  Lajkovi(
      {this.lajkoviId,
      this.korisnikId,
      required this.receptId,
      this.datumLajkanja,
      this.korisnik,
      this.recept});

  factory Lajkovi.FromJson(Map<String, dynamic> json) =>
      _$LajkoviFromJson(json);

  Map<String, dynamic> toJson() => _$LajkoviToJson(this);
}
